tests/runai_model_streamer_test/test_runai_utils.py

```diff
@@ -34,6 +36,26 @@ def test_runai_list_safetensors_local():
         assert len(safetensors) == len(files)
 
 
+def test_runai_pull_files_gcs():
+    os.environ["RUNAI_STREAMER_GCS_USE_ANONYMOUS_CREDENTIALS"] = "true"
+    model = ObjectStorageModel()
+    filename = "LT08_L1GT_074061_20130309_20170505_01_T2_MTL.txt"
+    gcs_bucket = "gs://gcp-public-data-landsat/LT08/01/074/061/LT08_L1GT_074061_20130309_20170505_01_T2/"
```

Comment:
Is that backet is guarenteed to be kept alive?
And do we want to couple it to vLLM?

Reply:
> We should use monkeypatch fixture to prevent this os.environ change from affecting other tests

Thanks for the suggestion. Updated.